src/screens/DataApi/DataApi.style.tsx

```diff
@@ -1,7 +1,28 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { MarkdownContent } from 'components/Markdown';
 import { COLOR_MAP } from 'common/colors';
 import theme from 'assets/theme';
+import LinkButton from 'components/LinkButton';
+
+const SharedButtonStyles = css`
```

Comment:
you can import this in from SectionButton.style instead of rewriting

Reply:
Do you think the `SectionButton` stuff should eventually be moved to a more common place? Feels more uncomfy to import directly from learn code (though I'm not sure I 100% follow that philosophy)

Reply:
buttons are soon to be redone anyway, so i guess its just fine to leave this all as is